marin/resources_utils.py

```diff
@@ -0,0 +1,70 @@
+
+# https://github.com/stanford-crfm/levanter/blob/162ef5f321cfc2e14b1b9c1652e2cffd42b395e2/src/levanter/utils/flop_utils.py#L40
+
+import logging
+logger = logging.getLogger("ray")
+
+# Base FLOPS for each device type (using mixed precision values where available)
+device_flops_map = {
+    # GPUs (using mixed precision values where available)
+    "H100": 1.513e15 / 2,  # amp_fp16/bf16; pcie
+    "A100": 312e12,  # amp_fp16/bf16
+    "A10G": 70e12,  # bf16: https://www.baseten.co/blog/nvidia-a10-vs-a10g-for-ml-model-inference/
+    "V100": 125e12,  # amp_fp16
+    "T4": 65e12,  # amp_fp16
+    "A6000": 309.7e12 / 2,  # bf16/fp16
+    
+    # TPU base FLOPS per chip,
+    "v4": 275e12,  # bf16, https://cloud.google.com/tpu/docs/v4
+    "v5litepod": 197e12,  # bf16, https://cloud.google.com/tpu/docs/v5e
+    "v5": 459e12,  # bf16, https://cloud.google.com/tpu/docs/v5
+    "v6e": 918e12,  # bf16, https://cloud.google.com/tpu/docs/v6e
```

Comment:
should these be "tpu-v4" or something for clarity?

Reply:
Just thinking that "v4" might be confusing out of context...  In general, I don't have a good idea of what the naming convention is (between what Ray uses, what Levanter uses, and what we have here).  (cc @dlwh )